react-components/src/analysis/view/AnalysesView.js

```diff
@@ -0,0 +1,955 @@
+/**
+ *  @author sriram
+ *
+ **/
+
+import React, { Component } from 'react';
+import { injectIntl } from "react-intl";
+
+import intlData from "../messages";
+import ids from "../ids";
+import exStyles from "../style";
+import appType from "../model/appType";
+import permission from "../model/permission";
+import Color from "../../util/CyVersePalette";
+import formatDate from "../../util/DateFormatter";
+import build from "../../util/DebugIDUtil";
+import withI18N, { formatMessage } from "../../util/I18NWrapper";
+
+import DotMenu from "./DotMenu";
+import AnalysisParametersDialog from "./dialogs/AnalysisParametersDialog";
+import AnalysisInfoDialog from "./dialogs/AnalysisInfoDialog";
+import analysisStatus from "../model/analysisStatus";
+import AnalysesToolbar from "./AnalysesToolbar";
+import DEPromptDialog from "../../util/dialog/DEPromptDialog";
+import ShareWithSupportDialog from "./dialogs/ShareWithSupportDialog";
+import DEHyperLink from "../../util/hyperlink/DEHyperLink";
+import DEConfirmationDialog from "../../util/dialog/DEConfirmationDialog";
+import EnhancedTableHead from "../../util/table/EnhancedTableHead";
+import TablePaginationActions from "../../util/table/TablePaginationActions";
+
+import Checkbox from "@material-ui/core/Checkbox";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TablePagination from "@material-ui/core/TablePagination";
+import TableRow from "@material-ui/core/TableRow";
+import { withStyles } from "@material-ui/core/styles";
+
+import LaunchIcon from "@material-ui/icons/Launch";
+import ListAltIcon from "@material-ui/icons/ListAlt";
+
+
+function AnalysisName(props) {
+    const name = props.analysis.name;
+    const isBatch = props.analysis.batch;
+    const className = props.classes.analysisName;
+    const handleGoToOutputFolder = props.handleGoToOutputFolder;
+    const handleBatchIconClick = props.handleBatchIconClick;
+    const interactiveUrls = props.analysis.interactive_urls;
+    const handleInteractiveUrlClick = props.handleInteractiveUrlClick;
+    const status = props.analysis.status;
+
+    if (isBatch) {
+        return (
+            <span className={className} onClick={handleGoToOutputFolder}>
+                <ListAltIcon onClick={handleBatchIconClick}
+                             style={{color: Color.darkGreen}}/>
+                <sup>{name}</sup>
+            </span>
+        );
+    } else if (status === analysisStatus.RUNNING && interactiveUrls && interactiveUrls.length > 0) {
```

Comment:
Also, regarding providing the VICE url during the `SUBMITTED` state - there are no backend changes needed, you actually only have to change this line so that the icon will display if it's in `SUBMITTED` or `RUNNING`.

Reply:
👍